master/internal/api_tasks.go

```diff
@@ -380,6 +382,28 @@ func (a *apiServer) TaskLogs(
 	})
 }
 
+func (a *apiServer) monitor(ctx context.Context, taskID model.TaskID, logs []*model.TaskLog) error {
```

Comment:
Found I bug I didn't notice: we don't want to monitor trial config. It always matches our log pattern. I will fix this. https://github.com/determined-ai/determined/blob/42535c22a320a715e6929ba6e3d25920c858e66c/harness/determined/exec/launch.py#L78-L81

Reply:
sounds good to me, I think we can do it as a follow up PR to this too, resolving this since we have the other thread